diff --git a/Startup/Playground.sh b/Startup/Playground.sh
index 4f37b18..e7b4b66 100755
--- a/Startup/Playground.sh
+++ b/Startup/Playground.sh
@@ -5,41 +5,41 @@ if [ -z ${MARTE_DIR+x} ]; then export MARTE_DIR=.; fi;
 
 
 LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/.
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/FunctionGeneratorGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/SimulinkInterfaceGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/TestGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/Raptor/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/equinoxGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/FFTGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/MathExpressionGAM
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/SpiderCalGAM
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/GAMs/PyGAM
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/FunctionGeneratorGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/SimulinkInterfaceGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/TestGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/Raptor/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/equinoxGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/FFTGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/MathExpressionGAM
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/SpiderCalGAM
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/GAMs/PyGAM
 LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/anaconda3/lib/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/DataSources/mcc118
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/DataSources/StreamOut
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/DataSources/StreamIn
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/DataSources/ConsoleOut
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/DataSources/MDSReaderNS
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/armv8-linux/Components/Interfaces/MDSEventManager
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_DIR/Build/armv8-linux/Core/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/EpicsDataSource/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/LinuxTimer/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/LoggerDataSource/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/NI6259/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/NI6368/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/SDN/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/FileDataSource/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/RealTimeThreadSynchronisation/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/RealTimeThreadAsyncBridge/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/UDP/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/GAMs/IOGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/GAMs/SSMGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/GAMs/WaveformGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/GAMs/BaseLib2GAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/GAMs/ConversionGAM/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/Interfaces/BaseLib2Wrapper/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/Interfaces/EPICSInterface/
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/armv8-linux/Components/DataSources/MDSWriter
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/DataSources/mcc118
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/DataSources/StreamOut
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/DataSources/StreamIn
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/DataSources/ConsoleOut
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/DataSources/MDSReaderNS
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTE_DIR/../Build/x86-linux/Components/Interfaces/MDSEventManager
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_DIR/Build/x86-linux/Core/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/EpicsDataSource/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/LinuxTimer/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/LoggerDataSource/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/NI6259/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/NI6368/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/SDN/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/FileDataSource/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/RealTimeThreadSynchronisation/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/RealTimeThreadAsyncBridge/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/UDP/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/GAMs/IOGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/GAMs/SSMGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/GAMs/WaveformGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/GAMs/BaseLib2GAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/GAMs/ConversionGAM/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/Interfaces/BaseLib2Wrapper/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/Interfaces/EPICSInterface/
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MARTe2_Components_DIR/Build/x86-linux/Components/DataSources/MDSWriter
 
 
 #Disable CPU speed changing
@@ -51,5 +51,5 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 #cgdb --args ../Build/linux/Startup/Playground.ex $1 $2 $3 $4
 #strace -o/tmp/strace.err ../Build/linux/Startup/Playground.ex $1 $2  $3 $4
 
-$MARTE_DIR/../Build/armv8-linux/Startup/Playground.ex $1 $2 $3 $4 
-#valgrind $MARTE_DIR/../Build/armv8-linux/Startup/Playground.ex $1 $2 $3 $4 
+$MARTE_DIR/../Build/x86-linux/Startup/Playground.ex $1 $2 $3 $4 
+#valgrind $MARTE_DIR/../Build/x86-linux/Startup/Playground.ex $1 $2 $3 $4 
